lib/linux_kernel/processor.c

```diff
@@ -644,6 +642,99 @@ M0_INTERNAL m0_processor_nr_t m0_processor_id_get(void)
 {
 	return smp_processor_id();
 }
+#elif defined CONFIG_AARCH64    /*CONFIG_AARCH64*/
```

Comment:
The comment /*CONFIG_AARCH64*/ is not needed.

Reply:
Removed